src/views/components/App.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+interface AppProps {
+    data: any
+}
+class App extends React.Component<AppProps> {
+    render(): any {
+        return (
+            <div>
+                <h1>Hello world</h1>
```

Comment:
why "hello world" ?

Reply:
This code doesn't exists anymore. See : https://github.com/ekino/expresso/pull/11